features/mask/core/src/main/java/org/apache/shardingsphere/mask/algorithm/cover/KeepFirstNLastMMaskAlgorithm.java

```diff
@@ -48,18 +50,32 @@ public void init(final Properties props) {
     }
     
     private Integer createFirstN(final Properties props) {
+        int firstN = 0;
         MaskAlgorithmPropsChecker.checkIntegerTypeConfig(props, FIRST_N, getType());
-        return Integer.parseInt(props.getProperty(FIRST_N));
+        String firstNValue = props.getProperty(FIRST_N);
+        if (!Strings.isNullOrEmpty(firstNValue)) {
+            firstN = Integer.parseInt(firstNValue);
+            ShardingSpherePreconditions.checkState(firstN > 0, () -> new MaskAlgorithmInitializationException(getType(), "first-n must be a positive integer."));
+        }
```

Comment:
Here can be extract the method to MaskAlgorithmPropsChecker, and it can be reused